src/app/auth/features/user-manager-detail/user-manager-detail.component.html

```diff
@@ -0,0 +1,96 @@
+@if (user(); as currentUser) {
+  <div class="container mx-auto p-6">
+    <div class="clearfix mb-6">
+      <button
+        mat-icon-button
+        (click)="goBack()"
+        aria-label="Go back to user list"
+        class="text-primary float-left mt-1 mr-4"
+      >
+        <mat-icon>chevron_left</mat-icon>
+      </button>
+      <h1 class="text-3xl font-bold">User: {{ currentUser.displayName }}</h1>
+    </div>
+
+    <div class="rounded-lg bg-white p-6 shadow-md">
+      <div class="grid grid-cols-1 gap-6 md:grid-cols-2">
+        <div>
+          <h2 class="mb-4 text-xl font-semibold">Basic Information</h2>
+          <div class="space-y-3">
+            <div>
+              <span class="block text-sm font-medium text-gray-700">Name</span>
```

Comment:
```suggestion
              <span class="block text-sm font-bold text-gray-700">Name</span>
```
The headers should be bolded for a better look. It's a bit too hard to parse as it stands right now

Reply:
I'm leaving this as is since this needs to be redesigned according to our wireframes anyway.